jbpm-workitem-itests/src/test/java/org/jbpm/workitem/springboot/samples/KafkaProxyAsyncSampleTest.java

```diff
@@ -138,7 +138,9 @@ public void testAsyncKafkaWIHNoConnection() throws Exception {
             
             startOtherProcess();
         } finally {
-            processService.abortProcessInstance(processInstanceId);
+            if (runtimeDataService.getProcessInstances(singletonList(STATE_ACTIVE), null, null).size() == 1) {
```

Comment:
why not invokeind abortPRocess just when the list is not empty
The other option is just wrap the call to abort between a try catch
I personally leave it as it was originally (if the abort calls fails, the test should fail)

Reply:
thanks, added an abortAllProcesses method as talked